main/webapp/modules/core/langs/translation-en.json

```diff
@@ -581,6 +581,11 @@
     "core-views/starred-rows": "Starred Rows",
     "core-views/facet-flag": "Facet by flag",
     "core-views/flagged-rows": "Flagged Rows",
+    "core-views/blank-rows": "Blank Rows",
+    "core-views/count-non-blank-values": "Count non-blank values/column",
```

Comment:
Would it be cleaner (and more consistent with the facet name) to write "values per column" instead of "values/column"?

Reply:
Yes, but the width of the menu (2nd level) seems to be fixed. The (longer) string "Count non-blank values per column" would cross the border line.

Reply:
You can drop the word "Count".  It's implied as other facets do.  Just say "Non-Blank values per column", etc.

Reply:
Alright, fixed with 2c266d3